src/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from "express";
+
+export const errorHandler = (
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  res.status ? res.status : res.status(500);
+  res.json({
```

Comment:
@maazjavaid you need to implement this on global level so when your `express-async-handler` throws error, your error handler listens to it and send it back to user.

Reply:
https://expressjs.com/en/guide/error-handling.html